package.json

```diff
@@ -21,50 +21,50 @@
   },
   "license": "MIT",
   "devDependencies": {
-    "babel-preset-es2015": "6.22.0",
-    "babel-preset-react": "6.23.0",
+    "babel-preset-es2015": "6.24.1",
+    "babel-preset-react": "6.24.1",
     "babelify": "7.3.0",
-    "browserify": "14.1.0",
-    "clean-css": "3.4.25",
+    "browserify": "14.4.0",
+    "clean-css": "3.4.27",
     "exorcist": "0.4.0",
     "flux": "3.1.2",
     "isomorphic-fetch": "2.2.1",
-    "mocha": "3.2.0",
-    "ngrok": "2.2.6",
-    "nock": "9.0.9",
+    "mocha": "3.4.2",
+    "ngrok": "2.2.11",
+    "nock": "9.0.13",
     "nodemon": "1.11.0",
-    "normalize.css": "5.0.0",
-    "react-dom": "15.4.2",
-    "react-router": "3.0.2",
-    "react": "15.4.2",
+    "normalize.css": "7.0.0",
+    "react-dom": "15.6.1",
+    "react-router": "3.0.5",
+    "react": "15.6.1",
     "supertest": "3.0.0",
     "uglifyify": "3.0.4",
     "ungrid": "1.0.1",
     "watchify": "3.9.0"
   },
   "dependencies": {
-    "async": "2.1.5",
-    "bl": "1.2.0",
-    "body-parser": "1.16.1",
+    "async": "2.5.0",
+    "bl": "1.2.1",
+    "body-parser": "1.17.2",
     "cradle": "0.7.1",
     "curry": "1.2.0",
-    "es6-object-assign": "1.0.3",
-    "es6-promise": "4.0.5",
+    "es6-object-assign": "1.1.0",
+    "es6-promise": "4.1.1",
     "expect.js": "0.3.1",
-    "express-session": "1.15.1",
-    "express-winston": "2.2.0",
-    "express": "4.14.1",
+    "express-session": "1.15.3",
+    "express-winston": "2.4.0",
+    "express": "4.15.3",
     "node-w3capi": "1.4.0",
     "nodemailer-mock-transport": "1.3.0",
-    "nodemailer-sendmail-transport": "1.0.0",
-    "nodemailer": "2.7.2",
+    "nodemailer-sendmail-transport": "1.0.2",
+    "nodemailer": "4.0.1",
```

Comment:
The [changelog of nodemailer](https://github.com/nodemailer/nodemailer/blob/master/CHANGELOG.md) seems to indicate that for v3+, the sendmail transport is included - do we still need `sendmail-transport` then?